frontend/public/module/k8s/pods.ts

```diff
@@ -234,11 +234,17 @@ export const podPhase = (pod: PodKind): PodPhase => {
   let phase = pod.status.phase || pod.status.reason;
 
   _.each(pod.status.initContainerStatuses, (container: ContainerStatus, i: number) => {
-    const { terminated, waiting } = container.state;
+    const { terminated, waiting, running } = container.state;
+    const initContainerSpec = pod.spec.initContainers.find((c) => c.name === container.name);
+
     if (terminated && terminated.exitCode === 0) {
       return true;
     }
 
+    if (running && container.started && initContainerSpec?.restartPolicy) {
```

Comment:
For clarity, I think it's better to have the explicit equality check and copy the order of the boolean checks from the original k8s code. I don't think we need to check both 'container.started' and 'container.status.running'. The k8s code only checks 'container.started'. 
```suggestion
    if (initContainerSpec?.restartPolicy === 'Always' && container.started) {
```